guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+
+Frequent backups are important for lightning wallets, as channel states frequently change and need to be updated.
+
+</div>
+
+
+
+Receivers using mobile wallets, which may not be online 24x7 will encounter some delay (once they come online) as their wallet scans the blockchain and performs calculations to detect their payments. Applications should communicate this fact during setup/onboarding. While the scanning itself is taking place, applications should show progress data and estimated time to complete the scanning process.
+
+The time to detect a specific received payment can be minimized if the sender & receiver can communicate with each other. Some applications already allow users to detect whether an address belongs to them. This feature should, if possible, be extended to include BIP-352 functionality.
+
+## Backup
+
+Like lightning wallets, on-chain bitcoin wallets supporting full-featured BIP-352 wallets require file backups. These backup files may contain the following information:
+backup seed (for BIP 39 wallets)
+- Scan and/or spend keys
+- labels & even contacts
+- actual txids?
+- wallet birthday
+
+{% include image-gallery.html pages = page.images_backup %}
+
+As a best practice, some of the above information such as seed, wallet birthday should be shown to the user in the UI itself in case they misplace the file or recovery application does not support backup files or simply a redundant manual backup.
+
+While simply backing up the seed is sufficient, this results in longer wallet recovery times and causes loss of useful information.
+
+> Since each silent payment output address is derived independently, regular backups are recommended.” - BIP352
+
+## Recovery
+
+A robust backup & recovery solution for BIP-352 wallets involves backup files since the wallet has important information besides just private key (recovery phrase), such as contacts and labels. Besides, wallets without a BIP-39 seed also need a backup file. On the other hand, BIP-39 wallets can be recovered with just a seed phrase, a process that can be sped up by entering the wallet creation date (wallet birthday). Thus, during wallet import (recovery), the application should provide multiple ways to restore their wallet and recover funds. This would include:
+- Backup file from cloud
+- import/upload backup file
+- enter recovery phrase (with or without wallet birthday)
+- Enter scan/spend key material
+
+These methods are similar to other wallet/address types including lightning wallets, and are explained here.
+
+{% include image-gallery.html pages = page.images_recovery %}
+
+## Pros and cons
```

Comment:
Everything below here seems like left-over from another page.

Reply:
You mean the recovery carousel?